### PR TITLE
feat: add dependencies policy

### PR DESCRIPTION
**Pull request by richzw**
Try to add dependencies policy per https://github.com/WasmEdge/WasmEdge/issues/2974

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

**Comment by alabulei1** [comment]
Hi @richzw 

Thanks for your contributions. Please fix DCO first.

**Comment by richzw** [comment]
> Hi @richzw
> 
> Thanks for your contributions. Please fix DCO first.

Sorry for missing the DCO before, the DCO was fixed. Could you please review it again? @alabulei1

File: SECURITY-INSIGHTS.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - github:juntao
+  - github:hydai
+  - github:q82419
+  - github:ibmibmibm
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  code-of-conduct: https://github.com/WasmEdge/WasmEdge/blob/master/docs/CODE_OF_CONDUCT.md
+documentation:
+- https://github.com/ossf/security-insights-spec/blob/main/README.md
```
Comment (alabulei1) [inline SECURITY-INSIGHTS.yml:21]: Should we use WasmEdge docs link here?

File: SECURITY-INSIGHTS.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+header:
+  schema-version: 0.13.4
+  last-updated: '2023-09-05'
+  last-reviewed: '2023-09-05'
+  expiration-date: '2024-09-05T01:00:00.000Z'
+  project-url: https://github.com/WasmEdge/WasmEdge
+  project-release: '0.13.4'
+project-lifecycle:
+  stage: active
+  bug-fixes-only: false
```
Comment (alabulei1) [inline SECURITY-INSIGHTS.yml:10]: Please add `roadmap` here.https://github.com/ossf/security-insights-spec/blob/main/specification.md#project-lifecycle

File: SECURITY-INSIGHTS.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+header:
+  schema-version: 0.13.4
+  last-updated: '2023-09-05'
+  last-reviewed: '2023-09-05'
+  expiration-date: '2024-09-05T01:00:00.000Z'
+  project-url: https://github.com/WasmEdge/WasmEdge
+  project-release: '0.13.4'
+project-lifecycle:
+  stage: active
```
Comment (alabulei1) [inline SECURITY-INSIGHTS.yml:9]: `stage` should be `status`.  https://github.com/ossf/security-insights-spec/blob/main/specification.md#project-lifecycle

File: SECURITY-INSIGHTS.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  bug-fixes-only: false
+  core-maintainers:
+  - github:juntao
+  - github:hydai
+  - github:q82419
+  - github:ibmibmibm
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  code-of-conduct: https://github.com/WasmEdge/WasmEdge/blob/master/docs/CODE_OF_CONDUCT.md
```
Comment (alabulei1) [inline SECURITY-INSIGHTS.yml:19]: We also need to add `automated-tools-list`. https://github.com/ossf/security-insights-spec/blob/main/specification.md#contribution-policy

File: SECURITY-INSIGHTS.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+documentation:
+- https://github.com/ossf/security-insights-spec/blob/main/README.md
+distribution-points:
+- https://github.com/WasmEdge/WasmEdge
```
Comment (alabulei1) [inline SECURITY-INSIGHTS.yml:23]: The URL should be WasmEdge's release page

**Review by alabulei1 — APPROVED** [review]

**Comment by alabulei1** [comment]
Hi @richzw 

I have added some comments based on the specification of Security Insights. Please check it out. Thanks. https://github.com/ossf/security-insights-spec/blob/main/specification.md#specification

In addition, Google OSS Fuzz adopts WasmEdge. Can we also add this to this doc. https://github.com/google/oss-fuzz/tree/master/projects/wasmedge

File: SECURITY-INSIGHTS.yml (hunk 1 above)
Comment (richzw) [reply SECURITY-INSIGHTS.yml:21]: Sorry for missing that

**Comment by alabulei1** [comment]
Hi  @richzw 

I think we should put Google Fuzzing on the Security Testing section.

https://github.com/ossf/security-insights-spec/blob/main/specification.md#security-testing

**Comment by richzw** [comment]
> Hi @richzw
> 
> I think we should put Google Fuzzing on the Security Testing section.
> 
> https://github.com/ossf/security-insights-spec/blob/main/specification.md#security-testing

Done, could you please review it again?

**Review by alabulei1 — APPROVED** [review]

**Comment by alabulei1** [comment]
LGTM!
@hydai Please merge this PR when you're free!

**Pull request merged**